java/src/jmri/configurexml/ConfigXmlManager.java

```diff
@@ -34,18 +34,22 @@
      * See the <a href="package-summary.html#schema">Schema versioning
      * discussion</a>. Also controls the stylesheet file version.
      */
-    static final public String schemaVersion = "-4-19-2";
+    static final public String schemaVersion = "-4-19-5";
```

Comment:
The added schema version should be removed, as `master` already has a schema that reads and writes the exiting content. If this change needs an addition, put it in the existing version.